payjoin/src/core/send/v2/session.rs

```diff
@@ -215,13 +208,43 @@ mod tests {
             persister.save_event(event).expect("In memory persister shouldn't fail");
         }
 
-        let (sender, session_history) =
-            replay_event_log(&persister).expect("In memory persister shouldn't fail");
-        assert_eq!(sender, test.expected_sender_state);
-        assert_eq!(session_history.fallback_tx(), test.expected_session_history.fallback_tx);
-        assert_eq!(*session_history.pj_param(), test.expected_session_history.pj_param);
-        assert_eq!(session_history.status(), test.expected_session_history.expected_status);
-        assert_eq!(session_history.terminal_error(), test.expected_error);
+        let session = match replay_event_log(&persister) {
```

Comment:
Can you explain why we removed the expect?

Reply:
perhaps there is some desire to modify the expiration test to use this helper that will result in an error?